CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Added
 
 ## Changed
+* Improved Router clear state program generation. ([#636](https://github.com/algorand/pyteal/pull/636))
```

Comment:
Could you also explain that this contains a breaking change with respect to Router clear states?

Reply:
Please grammar police [d943b3f](https://github.com/algorand/pyteal/pull/636/commits/d943b3f0fd126c8a6d571efeab93758314f3bf2a)